setup.py

```diff
@@ -46,11 +98,15 @@ def get_version():
     url='https://github.com/scikit-hep/probfit',
     package_dir={'probfit': 'probfit'},
     packages=['probfit'],
-    ext_modules=extensions,
+    ext_modules=get_extensions(),
+    setup_requires=[
+        'numpy',
+    ],
     install_requires=[
         'setuptools',
         'numpy',
-        'iminuit'
+        'iminuit',
+        'matplotlib==1.5.3'
```

Comment:
Why an exact version here? This will break pretty much any solve after pip 20.3 if anyone else does this. It certainly will not allow it to be installed into an environment with mplhep.